src/bin/common.rs

```diff
@@ -655,6 +655,9 @@ fn parse_config(matches: &ArgMatches<'_>) -> Result<EncoderConfig, CliError> {
   cfg.tile_cols = matches.value_of("TILE_COLS").unwrap().parse().unwrap();
   cfg.tile_rows = matches.value_of("TILE_ROWS").unwrap().parse().unwrap();
 
+  cfg.max_width = matches.value_of("MAX_WIDTH").unwrap().parse().unwrap();
+  cfg.max_height = matches.value_of("MAX_HEIGHT").unwrap().parse().unwrap();
+
```

Comment:
should this fail if max_width/max_height are < or > than width/height? (can't tell which sets of parameters determines the amount of pixels to be displayed)

Reply:
yes, less than. max_width and max_height are basically to set the sequence options so you can create a compatible sequence header for switch frames.

Reply:
added validation in Config::validate()